server/commands/read.go

```diff
@@ -35,25 +34,26 @@ func NewReadQuery(datastore storage.OpenFGADatastore, tracer trace.Tracer, logge
 	}
 }
 
-// Execute the ReadQuery, returning paginated `openfga.Tuple`(s) that match the tupleset
+// Execute the ReadQuery, returning paginated `openfga.Tuple`(s) that match the tuple. Return all tuples if the tuple is
+// nil or empty.
 func (q *ReadQuery) Execute(ctx context.Context, req *openfgapb.ReadRequest) (*openfgapb.ReadResponse, error) {
 	store := req.GetStoreId()
-	modelID := req.GetAuthorizationModelId()
 	tk := req.GetTupleKey()
 
+	// Restrict our reads due to some compatibility issues in one of our storage implementations.
```

Comment:
I don't understand this comment

Reply:
ah so it's meant to deter people from supporting other kinds of filters in the query?

Reply:
If we want to maintain wire compatibility with Auth0 FGA, then it is meant to cause us less pain at the moment 🙂